spring-cloud-contract-stub-runner/src/main/java/org/springframework/cloud/contract/stubrunner/StubRunnerExecutor.java

```diff
@@ -248,11 +248,22 @@ private void sendMessage(Contract groovyDsl) {
 		List<YamlContract> yamlContracts = yamlContractConverter.convertTo(Collections.singleton(groovyDsl));
 		YamlContract contract = yamlContracts.get(0);
 		setMessageType(contract, ContractVerifierMessageMetadata.MessageType.OUTPUT);
-		// TODO: Json is harcoded here
-		this.messageVerifierSender.send(
-				JsonOutput
```

Comment:
We would previously always convert to Json, with the new changes we are not doing this anymore. I think we shouldn't break that behaviour

Reply:
any action needed here? or is this just confirming the issue?

Reply:
Yeah, what I'm saying is that with the changes you've done we no longer default to JSON when there's just String. I think we should bring back that behaviour